python/cudf/cudf/utils/cudautils.py

```diff
@@ -76,25 +76,7 @@ def expand_mask_bits(size, bits):
 
 
 @cuda.jit
-def gpu_shift(in_col, out_col, N):
-    """Shift value at index i of an input array forward by N positions and
-    store the output in a new array.
-    """
-    i = cuda.grid(1)
-    if N > 0:
-        if i < in_col.size:
-            out_col[i] = in_col[i - N]
-        if i < N:
-            out_col[i] = 0
-    else:
-        if i <= (in_col.size + N):
-            out_col[i] = in_col[i - N]
-        if i >= (in_col.size + N) and i < in_col.size:
-            out_col[i] = 0
```

Comment:
Was this function no longer used?

Reply:
It was a stale code which wasn't being used by any APIs.